submissions/oluwabusolamiEluyela/html/index.html

```diff
@@ -0,0 +1,86 @@
+             <h2>Price: $20.82</h2>
+             <h4>Customers Ratings: 5.0  (110 Reviews)</h4>
+            </div>
+            <div>
+              <h2>Product description</h2>
+             <p style="color:purple;">
+                Written by well-respected authors, the Cambridge Checkpoint Mathematics suite provides a comprehensive structured resource 
+		which covers the full Cambridge Secondary 1 Mathematics framework in three stages. This brightly illustrated Coursebook for 
+		Stage 7 offers a comprehensive introduction to all topics covered in the syllabus. Worked examples show students how to tackle 
+		different problems, and plenty of exercise questions prepare students for the different types of question they will face in their 
+		exam. Coverage of the Problem Solving framework is integrated throughout the course, with questions relating to the framework 
+		statements highlighted in the Coursebook and Practice Book.
+              </p>
+            </div>
+          </li>
+          <hr />
+          <li>
+            <div>
+              <h2><a href="">Cambridge Global English Stage 7 Coursebook with Audio CD</a></h2>
+		<img src="CAMBRIDGE GLOBAL ENGLISH STAGE 7 COURSEBOOK WITH AUDIO CD.jpg" alt="CAMBRIDGE GLOBAL ENGLISH STAGE 7 COURSEBOOK WITH AUDIO CD" width="120" height="130"/>
+             <h2>Price: $36.32</h2>
+             <h4>Customers Ratings: 5.0  (115 Reviews)</h4>
+            </div>
+            <div>
+              <h2>Product description</h2>
+              <p style="color:purple;">
+                Cambridge Global English Stages 7-9 follow the Cambridge Lower Secondary English as a Second Language (ESL) Curriculum Framework. 
+		Coursebook 7 is organised into eighteen thematic units of study based on the Cambridge International English Scheme of Work for 
+		Stage 7. The topics and situations in Cambridge Global English have been selected to reflect this diversity and encourage learning 
+		about each other's lives through the medium of English. It presents realistic listening, speaking, reading and writing tasks, as 
+		well as end-of-unit projects similar to those students might encounter in the context of a first-language school. After every 
+		other unit, there is a literature spread, featuring authentic texts from a variety of sources. CEFR Level - B1
+              </p>
+            </div>
+        </div>
+        <hr />
+       <nav>
+        <ul>
+           <p><h3>Customers who viewed this item also viewed</h3>
+        <ol>
+          <a href="#">Cambridge Global English Workbook Stage 7</a></li>
+				<h5>$23.30</h5>
+	  	  
+	  <a href="#">Cambridge Global English Workbook Stage 8</a></li>
+				<h5>$23.30</h5>
+
+ 	  <a href="#">Cambridge Global English Teachers Resource Stage 7</a></li>
+				<h5>$40.50</h5>
+   
+          </li>
+        </ol>
+      </div>
+    </main>
+   <center><a href="#"><footer> Conditions of Use     Privacy Notice    &copy 2019-2020, Justify.com</footer></center>
```

Comment:
Place the anchor tag inside the footer tag instead. No need for the center tag. When you start doing CSS, you can centralize your work